config/cli/bookworm/main/config_cli_standard/packages.additional

```diff
@@ -20,6 +21,7 @@ libfuse2
 libnss-myhostname
 libpam-systemd
 libproc-processtable-perl
+netplan.io
```

Comment:
Bookworm uses netplan, is that correct?

Reply:
Not sure if they use it, but we added this about a year ago to both. I am just moving it from minimal image.